DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Maybe somewhere in here it would be useful to have more explanation of what will happen next (i.e. this is still some kind of external catalog that is uploaded to RSP for matching and other analysis, but now you're demonstrating how to grab it from some other server? Anyway some more general explanation of why this is useful could be helpful. Also, perhaps list 1-2 other examples of what catalogs could be accessed by PyVO?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Added some explanation</p>